src/pages/ProfilePage.js

```diff
@@ -62,13 +62,17 @@ const propTypes = {
         partnerUserID: PropTypes.string,
     }),
 
+    /** Indicated whether the report data is loading */
```

Comment:
I think this is clearer, since it seems this key isn't just used to indicate report data is loading. But all the data that openApp gets (including personalDetails). 

```suggestion
    /** Indicates whether the app is loading initial data */
```